include/sound/sof/header.h

```diff
@@ -114,7 +114,7 @@
 #define SOF_IPC_MESSAGE_ID(x)			((x) & 0xffff)
 
 /* maximum message size for mailbox Tx/Rx */
-#define SOF_IPC_MSG_MAX_SIZE			384
+#define SOF_IPC_MSG_MAX_SIZE			768
```

Comment:
I tried to recall or find where this restriction originally comes from, and I couldn't. @lgirdwood could you remind please? I'll make sure to make a note this time.
I think this restriction comes from the original IPC protocol from the alternative firmware implementation? Currently size limits seem to be set in the firmware by `MAILBOX_HOSTBOX_SIZE` / `MAILBOX_DSPBOX_SIZE` macros, for which the smallest value I found was 1024. Is this complete area usable as a mailbox or is a part of it used for other purposes? So, I'd go straight with the largest possible value here. Of course, it has to be reflected in the firmware too.

Reply:
@lyakh it was discussed on FW PR:
https://github.com/thesofproject/sof/pull/2216
Currently it cannot be set to the largest possible value but it will later with some changes to ipc msg allocations

Reply:
@abonislawski ok, got it, thanks

Reply:
how does this work if a new kernel using a max size of 768 is used with an old firmware relying on a value of 384?

And why not 1K while we've at it? or 4K as in the past?